src/bp/core/services/telemetry/sdk-methods.ts

```diff
@@ -0,0 +1,185 @@
+import { parse } from '@babel/parser'
+import traverse from '@babel/traverse'
+import LicensingService from 'common/licensing-service'
+import { buildSchema } from 'common/telemetry'
+import Database from 'core/database'
+import { TelemetryRepository } from 'core/repositories/telemetry'
+import { TYPES } from 'core/types'
+import { inject, injectable } from 'inversify'
+import _ from 'lodash'
+import ms from 'ms'
+
+import { GhostService } from '..'
+import { BotService } from '../bot-service'
+import { JobService } from '../job-service'
+
+import { BotHooks, getHooksLifecycle, Hook } from './hooks'
+import { TelemetryStats } from './telemetry-stats'
+
+const PARSE_CONFIG = { allowReturnOutsideFunction: true }
+
+interface UsageParams {
+  botId?: string
+  lifecycle?: string
+  enabled?: Boolean
+}
+
+interface Usage {
+  module: string
```

Comment:
more of a namespace than a module

Reply:
fixed: 64d2839d0b1bfe78162d88070543b062b7de4b9a